<?xml version="1.0" encoding="utf-8"?>
<report
	progress="">
	<activity
		result="Failed"
		duration="21.97s"
		durationms="21970"
		type="root"
		rid="27853dee069acbb"
		totalerrorcount="1"
		totalwarningcount="0"
		totalsuccesstestcasecount="0"
		totalfailedtestcasecount="0"
		totalblockedtestcasecount="0"
		childsuccesscount="0"
		childfailedcount="0"
		childblockedcount="0"
		totalsuccesscount="0"
		totalfailedcount="0"
		totalblockedcount="0"
		totalmaintenancemodecount="0"
		user="astan"
		rxversion="8.0.0.0"
		host="CLJSDEASTAN01"
		osversion="Windows 10 64bit"
		runtimeversion="4.0.30319.42000"
		procarch="32bit"
		oslanguage="en-US"
		dtnlanguage="ro-RO"
		screenresolution="3840x1080"
		timestamp="29.01.2018 13:42:25"
		timestampiso="2018-01-29T11:42:25"
		endtime="29.01.2018 13:42:47"
		timeoutfactor="1"
		module="CreateProject">
		<detail>
			CreateProject
		</detail>
		<varbindings>
		</varbindings>
		<item
			timeRelativeToTestSuiteStartTime="00:21.986"
			timeRelativeToTestModuleStartTime="00:21.986"
			timeWallClock="13:42:47"
			time="00:21.986"
			level="Error"
			category="Module">
			<message>
				The test run has been aborted by the user.
			</message>
			<metainfo
				loglvl="Error">
			</metainfo>
		</item>
	</activity>
</report>
